OgcApi.Net/Crs/CoordinateConverter.cs

```diff
@@ -29,5 +29,13 @@ public static double TileYToLat(int y, int z)
             var n = Math.PI - 2.0 * Math.PI * y / (1 << z);
             return 180.0 / Math.PI * Math.Atan(0.5 * (Math.Exp(n) - Math.Exp(-n)));
         }
+        public static double[] TileBounds(int x, int y, int z)
```

Comment:
Add an empty line before this one.
Consider returning Envelope object instead of an array of double.